lib/widgets/navigation_drawer.dart

```diff
@@ -193,6 +199,12 @@ class _ExpansionTile extends StatelessWidget {
             ),
           ),
           children: items.map((item) => Padding(padding: EdgeInsets.only(left: 28.0), child: item)).toList(),
+          onExpansionChanged: (isExpanded) {
+            if (isExpanded) Timer(Duration(milliseconds: 200), () =>
```

Comment:
1. Missing parenthesis for the if block.
2. Why do we need the timer here? The scroll is configured to animate in  400 milliseconds already.
3. Nice to have - Currently this logic won't work if we will put an expanded item in the top of the list (because there is a scroll to the end of the list). Better try to scroll for the current item to be visible.

Reply:
1. It is not required in this case, but I'll add it for readability.
2. The elements are not expanded yet when calling animateTo, that's why we wait for them to be visible.
3. True, I agree this is a specific and a quick solution to this story. We should implement a generic solution.

Reply:
Regarding #2 does it create a problem if we start the scroll animation before waiting for the expansion to complete?

Reply:
Yes, maxScrollExtent hasn't changed according to initial state, so it scrolls to the same position(does not move).

Reply:
> Yes, maxScrollExtent hasn't changed according to initial state, so it scrolls to the same position(does not move).

Got you.